diff --git a/dist/chunks/auth-plugin.cjs b/dist/chunks/auth-plugin.cjs
index beef77a9f480f095381f176b0da5a3c77c4ee25e..dff1216e3f4c6cacc2ba0e05d12254e56679e471 100644
--- a/dist/chunks/auth-plugin.cjs
+++ b/dist/chunks/auth-plugin.cjs
@@ -162,7 +162,7 @@ function getSession(req2, res2, appDir = false) {
 function getBlitzContext() {
   return __async$1(this, null, function* () {
     try {
-      const { headers, cookies } = eval("require('next/headers')");
+      const { headers, cookies } = require('next/headers');
       const req = new http.IncomingMessage(new net.Socket());
       req.headers = Object.fromEntries(headers());
       const csrfToken = cookies().get(index.COOKIE_CSRF_TOKEN());
@@ -201,7 +201,7 @@ function useAuthenticatedBlitzContext(_0) {
     const ctx = yield getBlitzContext();
     const userId = ctx.session.userId;
     try {
-      const { redirect } = eval("require('next/navigation')");
+      const { redirect } = require('next/navigation');
       if (userId) {
         debug("[useAuthenticatedBlitzContext] User is authenticated");
         if (redirectAuthenticatedTo) {
